core/src/operator.rs

```diff
@@ -329,14 +329,6 @@ where
         }
     }
 
-    /// Checks if utxo is valid, spendable by operator and not spent
-    /// Saves the utxo to the db
-    #[tracing::instrument(skip(self), err(level = tracing::Level::ERROR), ret(level = tracing::Level::TRACE))]
-    async fn set_funding_utxo(&self, funding_utxo: UTXO) -> Result<(), BridgeError> {
-        self.db.set_funding_utxo(None, funding_utxo).await?;
```

Comment:
I am not sure if we should delete this.

Reply:
Same as above. I can re-add them if needed.

Reply:
We should not delete this, indeed we should write tests that uses this

Reply:
Re added with a unit test.